controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -116,16 +117,24 @@ function fetch($request, $template = null, $display = false) {
 			$reviewsAvailable = false;
 			$submission = $this->getSubmission();
 			$reviewRound = $this->getReviewRound();
-			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /* @var $reviewAssignmentDao ReviewAssignmentDAO */
+			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /** @var ReviewAssignmentDAO $reviewAssignmentDao */
 			$reviewAssignments = $reviewAssignmentDao->getBySubmissionId($submission->getId(), $reviewRound->getId());
+
+			$reviewers = [];
+			/** @var ReviewAssignment $reviewAssignment */
 			foreach ($reviewAssignments as $reviewAssignment) {
 				if ($reviewAssignment->getDateCompleted() != null) {
 					$reviewsAvailable = true;
-					break;
+				}
+				if (in_array($reviewAssignment->getStatus(), [REVIEW_ASSIGNMENT_STATUS_COMPLETE, REVIEW_ASSIGNMENT_STATUS_RECEIVED, REVIEW_ASSIGNMENT_STATUS_THANKED])) {
```

Comment:
@asmecher I forgot to leave a comment here... Is this enough to consider it as a "completed" review or the previous line would be enough?

Reply:
I think it's best just like you have it.